src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -533,8 +533,13 @@ protected virtual void RegisterViewTypes()
 
 		protected virtual CGRect DetermineEmptyViewFrame()
 		{
-			return new CGRect(CollectionView.Frame.X, CollectionView.Frame.Y,
-				CollectionView.Frame.Width, CollectionView.Frame.Height);
+			nfloat emptyViewHeight = 0;
+
+			if (_emptyViewFormsElement is IView emptyView)
+			{
```

Comment:
@Vignesh-SF3580  this regressed when we don t have a layout as an empty view. I m created issue https://github.com/dotnet/maui/issues/26508 to track this. Do you have any ideas here? seems the height of a button is always 0 ?

Reply:
@rmarinho , The issue occurs because the WrapperView for the button is being set during the rendering phase in the SetupContainer method. However, when the DetermineEmptyViewFrame method is called to measure the button, the SizeThatFitsWrapper logic is triggered. At this point, the WrapperView has no subviews, resulting in the default size (0, 0) being returned. This is due to the fact that the measurement depends on the subview hierarchy, which is not yet fully initialized during this phase.

The PR created by you [PR #26513](https://github.com/dotnet/maui/pull/26513) effectively addresses this scenario.